stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -92,6 +95,19 @@ public void createToken(@NonNull final Card card, @NonNull final TokenCallback c
     }
 
     /**
+     * THe simplest way to create a {@link BankAccount} token. This runs on the default
+     * {@link Executor} and with the currently set {@link #defaultPublishableKey}.
+     *
+     * @param bankAccount the {@link BankAccount} used to create this token
+     * @param callback a {@link TokenCallback} to receive either the token or an error
+     */
+    public void createToken(
```

Comment:
It's actually super overloaded for historic reasons that predated me, and I haven't killed all the card versions because I don't want to break compatibility -- I only created two BankAccount versions. I don't want to change the older versions, but I could change the names of the newer ones.